data-node/src/test/java/org/graylog/datanode/testinfra/DatanodeDevContainerBuilder.java

```diff
@@ -43,6 +44,7 @@
 public class DatanodeDevContainerBuilder implements org.graylog.testing.datanode.DatanodeDevContainerBuilder {
```

Comment:
Could you name those classes differently, so that the same name does not cause confusion?

Reply:
rather in a separate PR, it's nothing new and the change is big already.